docs/DevGuide/Parallelization.md

```diff
@@ -175,24 +181,18 @@ Each %Parallel Component struct must have the following type aliases:
    there are no iterable actions in a phase then a `PhaseAction` need not be
    specified for that phase. However, at least one `PhaseAction`, even it if is
    empty, must be specified.
-4. `using options` is set to a (possibly empty) `tmpl::list` of the option
-   structs. The options are read in from the input file specified in the main
-   `Metavariables` struct. After being read in they are passed to the
-   `initialize` function of the parallel component, which is described below.
-   These options are typically only used during initialization.
-5. `using add_options_to_databox` is set to a struct that contains a static
-   `apply` function taking arguments `(db::DataBox<DbTagsList>&&, options...)`
-   where the `options...` are the arguments read from the input file using the
-   `options` type alias. Typically the `add_options_to_databox` will insert the
-   options into the `DataBox` for use during initialization. The
-   `add_options_to_databox` struct must also contain a type list named
-   `simple_tags` which contains a list of all the tags that the `apply` function
-   will add to the DataBox. No compute tags may be added at this stage. A
-   reasonable location for this struct is inside the action that will do the
-   initialization. Two reasonable defaults are provided,
-   `Parallel::AddNoOptionsToDataBox` and
-   `Parallel::ForwardAllOptionsToDataBox`.
-6. `using const_global_cache_tag_list` is set to a `tmpl::list` of OptionTags
+4. `using initialization_tags` which is a `tmpl::list` of all the tags
+   that correspond to items that will be inserted into the initial
+   `DataBox` of each component.  This list can usually be constructed
+   from the initialization actions of the component (i.e. the list of
+   actions in the `PhaseAction` list for the `Initialization` phase)
+   using the helper function `get_initialization_tags` (see the
+   examples of components below).  Each initialization action may
+   specify a type alias `using initialization_tags` which are a
+   `tmpl::list` of tags that will be fetched from the DataBox during
```

Comment:
I would've said `by the action` instead of `during the action`. I'll let you decide.